utils/wasm-builder/src/wasm_project.rs

```diff
@@ -171,11 +176,13 @@ impl WasmProject {
         fs::write(
             &wasm_binary_rs,
             format!(
-                r#"
-                    pub const WASM_BINARY: &[u8] = include_bytes!("{}");
-                    pub const WASM_BINARY_OPT: &[u8] = include_bytes!("{}");
-                    pub const WASM_BINARY_META: &[u8] = include_bytes!("{}");
-                "#,
+                r#"#[allow(dead_code)]
+pub const WASM_BINARY: &[u8] = include_bytes!("{}");
+#[allow(dead_code)]
+pub const WASM_BINARY_OPT: &[u8] = include_bytes!("{}");
+#[allow(dead_code)]
+pub const WASM_BINARY_META: &[u8] = include_bytes!("{}");
+"#,
```

Comment:
```suggestion

"#,
```

Reply:
No need for an empty new line as we already have a carriage return.